rcsbapi/data/schema.py

```diff
@@ -415,28 +420,65 @@ def __construct_query_networkx(self, input_ids, input_type, return_data_list):
         query = "query"
         return query
 
-    def __construct_query_rustworkx(self, input_ids: Dict[str, str], input_type, return_data_list):
+    def __construct_query_rustworkx(self, input_type, return_data_list, input_ids: Dict[str, str] = None, id_list=None):
+        if input_ids is None and id_list is None:
```

Comment:
I think this could be moved into construct query so it doesn't have to be repeated for networkx